src/crimson/osd/osd.cc

```diff
@@ -683,6 +689,24 @@ seastar::future<> OSD::_send_boot()
   m->metadata["osd_type"] = "crimson";
   return monc->send_message(std::move(m));
 }
+seastar::future<bool> OSD::_get_device_type()
+{
+  LOG_PREFIX(OSD::_get_device_type);
```

Comment:
this is repeating what _add_device_class does, you can give it a return value or make is_rotational a member of OSD instead

Reply:
done